### PR TITLE
ESR Closeout Preparation Catalog Task

### PR DESCRIPTION
**Pull request by sarwarahmed1**

### CODE REVIEW

File: src/test/java/ServiceNow/ESR/Playwright/Features/ESRNotification.feature [hunk 1]
```diff
@@ -13,4 +13,22 @@ Feature: ESR Notifications
...
+  @ESR-??? @sarwarahmed1 @In-Progress @playwright
+  Scenario: Test a watchlist update notification for the creation of an ESR-Q ticket
+    Given a user logs into Native View on the NCI at your service page
+    And navigates to create ESR
+    And fills out all required information for ESR-Q and clicks submit
+    And user clicks on Intake Preparation catalog task
+    And user fills out all required information for ESR-Q Intake Preparation
+    Then the user confirms a watchlist update notification is sent in the activity feed
+
+  @ESR-??? @sarwarahmed1 @In-Progress @playwright
```
Comment (hamidnesar) [inline src/test/java/ServiceNow/ESR/Playwright/Features/ESRNotification.feature:27]: There is no Story ID for these scenarios yet?

File: src/test/java/ServiceNow/ESR/Playwright/StepsImplementation/ESRTaskStepsImplementation.java [hunk 2]
```diff
@@ -150,4 +150,68 @@ public static void completeEngineeringProjectExecutionCatalogTask() {
...
+    public static void completeCloseoutPreparationCatalogTaskForESRQ() {
+        Playwright_Common_Locators.iframeLocator().locator("#tabs2_list").getByText("Catalog Tasks (4)").click();
+        Playwright_Common_Locators.iframeLocator().locator("(//td[@class='vt']/a)[1]").click();
+        Playwright_Common_Locators.iframeLocator().locator("//input[@aria-labelledby='label.sc_task.assigned_to']").click();
+        Playwright_Common_Locators.iframeLocator().locator("//input[@aria-labelledby='label.sc_task.assigned_to']").fill("CBIIT Test Account");
+        Playwright_Common_Locators.iframeLocator().getByRole(AriaRole.ROW, new FrameLocator.GetByRoleOptions().setName("Actual Start Date  Choose")).getByRole(AriaRole.LINK).first().click();
+        Playwright_Common_Locators.iframeLocator().getByLabel("Go to Today", new FrameLocator.GetByLabelOptions().setExact(true)).click();
+
+        Playwright_Common_Locators.iframeLocator().locator("(//a[@class=' btn btn-default btn-ref'])[5]").click();
+
```
Comment (hamidnesar) [inline src/test/java/ServiceNow/ESR/Playwright/StepsImplementation/ESRTaskStepsImplementation.java:166]: Is there any reason for having these empty lines 164 and 166?

**Review by hamidnesar — COMMENTED** [review]
Hi, 
Please review the comments. Thanks!

File: src/test/java/ServiceNow/ESR/Playwright/Features/ESRNotification.feature (hunk 1 above)
Comment (sarwarahmed1) [reply src/test/java/ServiceNow/ESR/Playwright/Features/ESRNotification.feature]: Forgot to put them, fixed!

**Review by Mariachaudhry — APPROVED** [review]
Approved!

File: src/test/java/ServiceNow/ESR/Playwright/StepsImplementation/ESRTaskStepsImplementation.java (hunk 2 above)
Comment (sarwarahmed1) [reply src/test/java/ServiceNow/ESR/Playwright/StepsImplementation/ESRTaskStepsImplementation.java]: Removed!

**Review by hamidnesar — APPROVED** [review]
Approved!

**Review by katm007 — APPROVED** [review]
Approved!

**Pull request merged**